packages/storybook-vue/stories/components/telekom-brand-header-navigation/TelekomBrandHeader.stories.mdx

```diff
@@ -108,6 +108,7 @@ export const TemplateProfileMenu = (args, { argTypes }) => ({
               :login-settings-url="loginSettingsUrl"
               :hide-login-settings="hideLoginSettings"
               :logout-url="logoutUrl"
+              :type="type"
```

Comment:
Remove this since the 'standard' is not a type. it fixes the build if you remove this